PRESUBMIT.py

```diff
@@ -280,6 +280,48 @@ def CheckLicense(input_api, output_api):
     return result
 
 
+def CheckNewSourceFileWithoutGnChangeOnUpload(input_api, output_api):
+    """Checks newly added .cc/.h/.mm files have corresponding GN changes."""
+    files_to_check = (
+        r'.+\.cc$',
+        r'.+\.h$',
```

Comment:
`.h` files already validated by the existing upstream check (we do run it). Let's validate only `.cc, .c, .mm, .m` in our check.